src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1278,8 +1278,14 @@ class ApplicationManagerImpl
    * @param to the new HMILevel for the certain app.
    */
   void ProcessApp(const uint32_t app_id,
-                  const mobile_apis::HMILevel::eType from,
-                  const mobile_apis::HMILevel::eType to);
+                  const HmiStatePtr from,
+                  const HmiStatePtr to);
+
+  /**
+   * @brief Starts EndStream timer for a specified application
+   * @param app_id Application to process
+   */
+  void StartEndStreamTimer(uint32_t app_id);
```

Comment:
const?

Reply:
@mked-luxoft fixed in 4f71e8ac39d897fdc5afcaef8e44c504fbda9ef3